examples/hotrod/README.md

```diff
@@ -44,3 +44,29 @@ Then open http://127.0.0.1:8080
 
 [hotrod-tutorial]: https://medium.com/@YuriShkuro/take-opentracing-for-a-hotrod-ride-f6e3141f7941
 [hotrod-openshift]: https://blog.openshift.com/openshift-commons-briefing-82-distributed-tracing-with-jaeger-prometheus-on-kubernetes/
+
+
+
+# HotROD on Containerized Version
+## Requirements
+- go
+- docker-compose
+- docker
+### Creating the images
+
+```
+chmod +x ./create-image.sh
+./create-image.sh
```

Comment:
`x` flag can be set and recorded in git. If not, the instruction can be `sh ./create-image.sh`